code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -856,3 +856,30 @@
 	strip_delay = 130
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	actions_types = list()
+
+// Doomguy ERT version
+/obj/item/clothing/suit/space/hardsuit/shielded/doomguy
+	name = "juggernaut armor"
+	desc = "A somehow spaceworthy set of armor with outstanding protection against almost everything. Comes in an oddly nostalgic green. "
+	icon_state = "doomguy"
+	item_state = "doomguy"
+	hardsuit_type = "syndi"
+	max_charges = 1
+	current_charges = 1
+	recharge_delay = 100
+	armor = list("melee" = 90, "bullet" = 90, "laser" = 70, "energy" = 70, "bomb" = 100, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```

Comment:
You might as well crank these values up to something absurd like 200, to make armor-piercing weapons less effective.

Don't worry, effective armor values (after armor_penetration is applied) cap out at 90, so you're not gonna make Doomguy immune to almost every attack in the game or something.

Reply:
I will do that!